### PR TITLE
Feature: Home Container Assignment

### PR DESCRIPTION
**Pull request by urvi04s**
Home Container Assignment(3)

Desktop View:
Screenshot 2024-07-23 at 3 08 16 PM

Mobile View:
Screenshot 2024-07-23 at 3 06 11 PM

### CODE REVIEW

File: Assets/scss/components/_home-container.scss [hunk 1]
```diff
@@ -0,0 +1,129 @@
+.home-container {
+  background-image: url("../../images/hero-container-background.jpg");
```
Comment (Lalit-gour1) [inline Assets/scss/components/_home-container.scss:2]: @urvi04s Please use image that is already in assets according to design and don't render image as background-image, render it as img tag or store the value of background-image value in a css variable in inline style then use it. So later on when we'll integrate into AEM then it'll be easy.
Cc:- @mathrik  @msharmaRP @jvaishnavrp

**Comment by Lalit-gour1** [comment]
@urvi04s  Please resolve conflict too

File: Assets/scss/components/_home-container.scss (hunk 1 above)
Comment (urvi04s) [reply Assets/scss/components/_home-container.scss]: Done thanks for the suggestion

**Review by mathrik — APPROVED** [review]
This looks solid to me

**Comment by mathrik** [comment]
There are new conflicts you'll need to resolve

**Pull request merged**